src/script/interpreter.cpp

```diff
@@ -335,9 +335,70 @@ bool EvalScript(vector<vector<unsigned char> >& stack, const CScript& script, un
                 // Control
                 //
                 case OP_NOP:
-                break;
+                    break;
+
+                case OP_CHECKLOCKTIMEVERIFY:
+                {
```

Comment:
Well, another blocker is the new softfork deployment mechanism @sipa is working on.
And remember this doesn't have to be part of 0.11, we could just release 0.11.1 with this a week after releasing 0.11.
So, yes, I think we have time for an identical-build fixup commit (won't imply more testing), why clean later when you can avoid it in the first place?